.travis.yml

```diff
@@ -7,7 +7,9 @@ before_script:
  - chmod +x deploy.sh
 install:
   - pip3 install -r requirements.txt
+  - pip3 install pylint
```

Comment:
by having pylint in the `requirements.txt` do you need this line, installing pylint?
I suggest removing this. what do you think? @daksha-aeer

Reply:
Done!